ynr/apps/elections/uk/every_election.py

```diff
@@ -141,7 +136,9 @@ def get_or_create_post(self):
                 label = self["organisation"]["official_name"]
                 role = self["elected_role"]
             try:
-                self.post_object = Post.objects.get(slug=slug)
+                self.post_object = Post.objects.get(
+                    slug=slug, organization=self.organization_object
+                )
```

Comment:
In this I'm:

1. assuming `self["division"]["official_identifier"]` is always unique
2. also using the organisation in the look up. 

I expect I don't need to do both of these things, but making sure we have the correct organisation feels like the right thing to do here anyway. 

There is a unique constraint on `Post.slug`, but not `unique_together('slug', 'organization'). I should move to that?